SocietalConstructionToolTests/Snapshots/ParserTests/BuiltinFunctions.verified.txt

```diff
@@ -1 +1 @@
-﻿(start (class_def class Foo ( args_def ) { (class_body (state state Bar { statement_list })) }) (function function f ( args_def ) -> (type void) { (statement_list (statement (expression_statement (call_expression exists ( (args_call (expression (agent_predicate Foo :: Bar ( args_agent )))) ))) ;) (statement (expression_statement (call_expression count ( (args_call (expression (agent_predicate Foo :: Bar ( args_agent )))) ))) ;) (statement (expression_statement (call_expression rand ( args_call ))) ;) (statement (expression_statement (call_expression seed ( (args_call (expression (literal 123))) ))) ;)) }) <EOF>)
\ No newline at end of file
+﻿(start (class_def class Foo ( args_def ) { (class_body (state state Bar { (statement_list (statement (enter enter Bar <missing ';'>))) })) }) (function function f ( args_def ) -> (type void) { (statement_list (statement (expression_statement (call_expression exists ( (args_call (expression (agent_predicate Foo :: Bar ( args_agent )))) ))) ;) (statement (expression_statement (call_expression count ( (args_call (expression (agent_predicate Foo :: Bar ( args_agent )))) ))) ;) (statement (expression_statement (call_expression rand ( args_call ))) ;) (statement (expression_statement (call_expression seed ( (args_call (expression (literal 123))) ))) ;)) }) <EOF>)
```

Comment:
Fix syntax error in test file

Reply:
![image](https://github.com/DATP4G6/sct/assets/25910953/deeba288-6891-4b2f-b43c-e6e880ec9b91)